tyr/servers/server.py

```diff
@@ -140,8 +163,9 @@ def configure(self):
         self.establish_route53_connection()
 
         if self.ami is None:
-            self.log.warn('No AMI provided')
-            self.ami = 'ami-8fcee4e5'
```

Comment:
This should still be the default fall back. The default value of `self.ami` is `None`, removing this as the fall back will require developers to either always provide this AMI or to  always use `get_latest_ami`.